caktus/Traffic-Stops
# Cu 47eg9mr create a resource tab with policy data

- Add `Resource` model
- Add views + serializers for resources
- Add Resource component for frontend

To test:
- Go to the resources tab in the admin and create a new resource for an agency
- Then go to the frontend and view the new tab in the navbar for Resources
- You should then see the details for that resources listed.
